src/app/pages/Itinerary/Itinerary.tsx

```diff
@@ -2,11 +2,25 @@ import React from 'react';
 import DestinationCard from '../../components/DestinationCard/DestinationCard';
 import type { DestinationType } from '../../utils/DestinationType';
 import NavigationButton from '../../components/NavigationButton/NavigationButton';
+import ItineraryMap from '../../components/ItineraryMap/ItineraryMap';
+import type { LngLatLike } from 'mapbox-gl';
 
 export default function Itinerary(): JSX.Element {
   const Itinerary = JSON.parse(
     window.localStorage.getItem('itinerary') || '[]'
   );
+  const locationsCoordinates: LngLatLike[] = [];
+
+  {
+    Itinerary.map((destination: DestinationType) =>
+      locationsCoordinates.push(
+        destination.location.geometry.coordinates as LngLatLike
+      )
+    );
+  }
+
+  console.log(locationsCoordinates);
```

Comment:
I guess you can delete this and the empty lines around.

Reply:
Thanks for spotting!